db/select.sql

```diff
@@ -0,0 +1,5 @@
+\c be_review_nc_news_test
+
+\dt
+
+select * from articles
```

Comment:
If this is a file exclusively for you to 'play with queries', I would .gitignore it